packages/dds/tree/src/test/shared-tree/fuzz/anchorStability.fuzz.spec.ts

```diff
@@ -73,6 +64,8 @@ describe("Fuzz - anchor stability", () => {
 	const opsPerRun = 20;
 	const runsPerBatch = 50;
 	describe("Anchors are unaffected by aborted transaction", () => {
+		// TODO: Currently manually disposing the view when applying the schema op is causing a double dispose issue.
```

Comment:
you should link https://dev.azure.com/fluidframework/internal/_workitems/edit/11436

Reply:
Updated work item, and added comments to fuzz tests linking to the work item